stanford_soe_helper.module

```diff
@@ -5,3 +5,45 @@
  * Code for the Stanford SOE Helper module.
  */
 
+/**
+ * Implements hook_init().
+ */
+function stanford_soe_helper_init() {
+
+  // Add css files
+  $file_paths = array(
+    drupal_get_path('module', 'stanford_soe_helper') . "/css/soe_helper.css",
+    drupal_get_path('module', 'stanford_soe_helper') . "/css/stanford_news_layouts.css"
+  );
+
+  $weight = 9999;
+  $conditions = "admin
+admin*
+admin/*
+node/*/edit
+block/*/edit*
+block/add*
+node/add*
+user/*/edit
+node/*/delete";
+
+  $css_rules = array();
+  $css_rules['rule_conditions'] = $conditions;
+  $css_rules['rule_type'] = CSS_INJECTOR_PAGES_NOTLISTED;
+
+  $pass = _css_injector_evaluate_rule($css_rules);
```

Comment:
we should add a dependency for css_injector to this module because it is using _css_injector_evaluate_rule